src/data/gurbaninow/types.ts

```diff
@@ -0,0 +1,78 @@
+type SourcePartial = {
```

Comment:
is this called partial because the values are optional?

Reply:
No, because it's a reusable component of other types.